src/test/java/com/salesforce/dynamodbv2/mt/mappers/MtAmazonDynamoDbTestRunner.java

```diff
@@ -485,93 +485,93 @@ void run() {
         log.info("END test " + testDescription);
     }
 
-    private List<MTRecord> getExpectedMTRecords() {
+    private List<MtAmazonDynamoDb.MtRecord> getExpectedMtRecords() {
         return ImmutableList.of(
-            new MTRecord()
+            new MtAmazonDynamoDb.MtRecord()
```

Comment:
why is this expanded to explicitly denote the outer class (i.e., "MtAmazonDynamoDb.MtRecord" instead of just "MtRecord")?